library/azure_rm_devtestlabslab.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabslab
+version_added: "2.8"
+short_description: Manage Azure Lab instance.
+description:
+    - Create, update and delete instance of Azure Lab.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    name:
+        description:
+            - The name of the lab.
+        required: True
+    location:
+        description:
+            - The location of the resource.
+    lab_storage_type:
+        description:
+            - Type of storage used by the lab. It can be either C(premium) or C(standard). Default is C(premium).
+        choices:
+            - 'standard'
+            - 'premium'
+    premium_data_disks:
+        description:
```

Comment:
I(lab_storage_type) is C(enabled), then ....

Reply:
fixed